react-native/android/app/src/main/java/io/keybase/android/KeyStore.java

```diff
@@ -28,7 +28,7 @@
 import go.keybase.Keybase;
 import io.keybase.android.keystore.KeyStoreHelper;
 
-public class KeyStore extends Keybase.ExternalKeyStore.Stub {
+public class KeyStore implements Keybase.ExternalKeyStore {
```

Comment:
gomobile's api changed and I only found out because the CI was building a different version. :disappointed: